vendor/assets/pdf_forms/et3-v1-en.yml

```diff
@@ -1,12 +1,13 @@
+---
```

Comment:
This file was re generated by my rake task (mentioned earlier) and tweaked accordingly - hence why strings that used to be quoted no longer are - as they didnt need to be.